RESOURCES/INTHEWILD.md

```diff
@@ -113,6 +113,7 @@ Join our growing community!
 - [Whale](http://whale.im)
 - [Windsor.ai](https://www.windsor.ai/) [@octaviancorlade]
 - [Zeta](https://www.zeta.tech/) [@shaikidris]
+- [Caizin](https://caizin.com/)
```

Comment:
This list is alphabetically sorted, please move it to the appropriate location. Also, feel free to add your GitHub handle after the link (see other similar entries)